packages/editor-ui/src/modules/settings.ts

```diff
@@ -131,6 +131,15 @@ const module: Module<ISettingsState, IRootState> = {
 				return e;
 			}
 		},
+		async skipOwnerSetup(context: ActionContext<ISettingsState, IRootState>) {
+			try {
+				context.commit('stopShowingSetupPage');
```

Comment:
was this needed? we are already doing this in app.vue as soon as we load page for the first time. we should remove that from here or there.

Reply:
open for discussion: I think the state change should happen on the action and needs to be checked in the app as well for redirection. However, both actions fulfill different reasons. Skip to really set it has been skipped and App to not  go there again on refresh.

Happy to remove it if you prefer 👍🏼

Reply:
either or. here makes more sense. but we did not have this logic before... so you need to remove it from `App.vue` and add it to owner creation.